### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/KazuCocoa/appium-uiautomator2-server/pull/1/files

I got a PR from @vmaxim and this PR is it to appium/appium-uiautomator2-server .
(Sorry for my notice after merging https://github.com/appium/appium-uiautomator2-server/pull/133 )

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/model/Session.java [hunk 1]
```diff
@@ -33,15 +34,17 @@ public String getSessionId() {
...
         }
         return shouldUseCompactResponses;
     }
 
     public static String[] getElementResponseAttributes() {
-        if (Session.capabilities.containsKey(CAP_ELEMENT_RESPONSE_ATTRIBUTES)) {
-            return Session.capabilities.get(CAP_ELEMENT_RESPONSE_ATTRIBUTES).toString().split(",");
+        if (Session.capabilities.containsKey(ELEMENT_RESPONSE_ATTRIBUTES.toString())) {
+            return Session.capabilities.get(ELEMENT_RESPONSE_ATTRIBUTES.toString()).toString().
+                    split(",");
```
Comment (mykola-mokhnach) [inline app/src/main/java/io/appium/uiautomator2/model/Session.java]: in java the dot separator is usually the first character if wrapped to a new string:

```
.toString()
.split(",")
```

File: app/src/main/java/io/appium/uiautomator2/model/settings/AbstractSetting.java [hunk 2]
```diff
@@ -22,35 +22,45 @@
...
     }
 
+    public abstract T getValue();
+
     protected abstract void apply(T value);
 
     private T convertValue(Object value) {
-        if (!valueType.isInstance(value)) {
+        try {
+            if (valueType == Long.class && value instanceof Number) {
```
Comment (mykola-mokhnach) [inline app/src/main/java/io/appium/uiautomator2/model/settings/AbstractSetting.java]: I assume it is obvious than if the first condition here is true then the second one is true as well.

Comment (vmaxim) [reply app/src/main/java/io/appium/uiautomator2/model/settings/AbstractSetting.java]: Can you explain your point in more detail? I can't find any correlation between these two conditions.

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/model/settings/AbstractSetting.java]: I mean any value of Long class is a number instance

Comment (vmaxim) [reply app/src/main/java/io/appium/uiautomator2/model/settings/AbstractSetting.java]: ```valueType``` represents the type of setting, but ```value``` is the desired value. Therefore these variables can belong to different classes.

**Pull request merged**